.eslintrc.js

```diff
@@ -32,7 +33,10 @@ module.exports = {
     "plugin:mozilla/recommended", // require("eslint-plugin-mozilla")
     "plugin:mozilla/browser-test",
     "plugin:mozilla/mochitest-test",
-    "plugin:mozilla/xpcshell-test"
+    "plugin:mozilla/xpcshell-test",
+    "prettier",
```

Comment:
Note to self: I think this is superfluous and wants to go away.

Reply:
Although maybe not: https://github.com/prettier/eslint-config-prettier#installation